mining/connecting-to-a-remote-headless-miner.md

```diff
@@ -1,8 +1,56 @@
 # Connecting to a remote \(headless\) miner
 
-{% hint style="info" %}
-This article is a stub. Maybe you can help write it!
-{% endhint %}
+To connect to a miner and not having to turn off the browsers SSL/TLS you can:
+- issue a Certificate with a Domain you own and setup a proxy (comlicated)
+- port worward the miner to your localhost
 
+## Port forward with ssh
 
+SSH is a way to access remote machines and allows port forwarding.
+To set this up there are a few simple steps:
 
+### 1. Installing ssh
+
+SSH is already installed on GNU/Linux.
+For Windows install your favorite application.  
+
+### 2. Editing `~/.ssh/config`
+
+This step is done so that you can ssh into machines without typing commandflags each time.
```

Comment:
```suggestion
This step is done so that you can ssh into machines without typing commandflags each time. Replace the hostname,  user, and `localhost:8000` with your port 
```